app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html

```diff
@@ -179,6 +242,11 @@ <h2 class="heading-medium">@messages("task-list.acquisition-details.link")</h2>
           <h2 class="heading-medium">@messages("task-list.relief-details.link")</h2>
           @reliefDetailsSummary(s.reliefDetails, subscribedDetails.isATrust, representativeType)
 
+          @if(glarAmount.isDefined && isFurtherReturn.contains(true)){
```

Comment:
e.g. here you could just do:
```scala
s.gainOrLossAfterReliefs.map{ g =>
  ...
}
```
then you wouldn't have to do `.getOrElse`

Reply:
fixed